translations/zh_CN.ts

```diff
@@ -956,6 +960,14 @@ This entry is ignored by V2Ray core when using DoH servers.</source>
         <source>Disable System Proxy</source>
         <translation>禁用系统代理</translation>
     </message>
+    <message>
+        <source>Enable Bypass CN Mainland</source>
+        <translation>启用绕过中国大陆</translation>
+    </message>
+    <message>
+        <source>Disable Bypass CN Mainland</source>
+        <translation>禁用绕过中国大陆</translation>
+    </message>
```

Comment:
You need to commit the changes to en_US.ts instead of zh_CN.ts

Reply:
I'm sorry but I don't understand how the translation mechanism works. I'll commit a new PR without translation to the dev branch. Could you kindly give me some directions on making translation?

Reply:
I have no idea, it may be that my system language is English and it automatically generates English.